nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/awsfulltests.yml

```diff
@@ -0,0 +1,36 @@
+name: nf-core AWS full size tests
+# This workflow is triggered on push to the master branch.
+# It runs the -profile 'test_full' on AWS batch
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types: [published]
+
+jobs:
+  run-awstest:
+    name: Run AWS tests
+    if: github.repository == '{{ cookiecutter.name }}'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Miniconda
+        uses: goanpeca/setup-miniconda@v1.0.2
+        with:
+          auto-update-conda: true
+          python-version: 3.7
+      - name: Install awscli
+        run: conda install -c conda-forge awscli
+      - name: Start AWS batch job
+        env:
+          AWS_ACCESS_KEY_ID: {% raw %}${{ secrets.AWSTEST_KEY_ID }}{% endraw %}
+          AWS_SECRET_ACCESS_KEY: {% raw %}${{ secrets.AWSTEST_KEY_SECRET }}{% endraw %}
+          TOWER_ACCESS_TOKEN: {% raw %}${{ secrets.AWSTEST_TOWER_TOKEN }}{% endraw %}
+        run: |
+          aws batch submit-job \
+            --region eu-west-1 \
+            --job-name nf-core-{{ cookiecutter.name_noslash }} \
+            --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' \
```

Comment:
I wonder if we put this in an organisation secret too. Not because it’s secret, more so that we can change it without updating pipelines.

Same for the s3 bucket names etc. Might be nice.

Reply:
Ah yes, makes sense!